docs/index.md

```diff
@@ -60,34 +65,158 @@ resource "ocm_group_membership" "my_admin" {
 
 The value of the `token` attribute of the provider should be the OCM
 authentication token that you can get [here](https://console.redhat.com/openshift/token).
-If this attribute isn't used then the provider will try to get the token from
+If this attribute isn't used then the provider will try to get the token it from
 the `OCM_TOKEN` environment variable.
 
-## Schema
+### Create AWS Rosa STS Cluster
+
+The following example shows a production grade rosa cluster with:
+
+* Existing VPC & Subnets
+* Multi AZ
+* Proxy
+* STS
+
+```
+data "aws_caller_identity" "current" {}
+
+locals {
+  sts_roles = {
+      role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Installer-Role",
+      support_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Support-Role",
+      operator_iam_roles = [
```

Comment:
You only need the operator_role_prefix
Please see the example:  https://github.com/terraform-redhat/terraform-provider-ocm/blob/main/examples/create_rosa_cluster/create_rosa_sts_cluster/cluster/main.tf#L36